docs/index.html

```diff
@@ -2501,7 +2501,8 @@ <h4 id="_math_functions"><a class="link" href="#_math_functions">Math Functions<
 <tbody>
 <tr>
 <td class="tableblock halign-left valign-top"><p class="tableblock"><code>apoc.math.round(value,[precision=0],mode=[HALF_UP,CEILING,FLOOR,UP,DOWN,HALF_EVEN,HALF_DOWN,DOWN,UNNECESSARY])</code></p></td>
-<td class="tableblock halign-left valign-top"><p class="tableblock">rounds value with optionally given precision (default 0) and optional rounding mode (default HALF_UP)</p></td>
+<td class="tableblock halign-left valign-top"><p class="tableblock">Rounds value with optionally given precision (default 0) and optional rounding mode (default HALF_UP).
```

Comment:
I think that instead of updating this (btw it should be generated, but I could be wrong) we need to update also `docs/asciidoc/modules/ROOT/pages/overview/apoc.math/apoc.math.round.adoc`

Reply:
I thought that might be the wrong place, but the file you are linking to has a comment on top saying `This file is generated by DocsTest, so don't change it!` so I don't think that is right either

Reply:
I think that is the right place because it affects the function, you can try to regenerate it by executing `DocsTest`. But maybe @JMHReif knows a better place!

Reply:
I think I found a better, not generated place to add it now. It looks like it will show up as a note in the end of this page now: https://neo4j.com/labs/apoc/4.1/overview/apoc.math/apoc.math.round/

Reply:
It's not the same page that I shared with you?

Reply:
Oh you mean the usage one! I see!

Reply:
Yes that one will be included in the page you shared, could see that when I ran the DocsTest